front_end/connect-to-car.html

```diff
@@ -13,13 +13,25 @@
   <div class="back-button-container">
     <a href="mainMenu.html"><img class="back-button" src="img/back-button.png" alt="button"></a>
   </div>
-  
+
   <div class="dynamic_status_information">
-   <p class="status", id="speed">Speed: 0m/s</p>
-   <p class="status", id="angle">Angle: 0</p>
-   <p class="status", id="distance">Distance: 0 m</p>
-   <p class="status", id="charge">Charge: 100%</p>
-  </div>
+    <div class="help_status">
+      <p class="status", id="speed"><u>Speed: 0m/s</u></p>
+      <span class="status_help">Controls: W,A,S,D</span>
+    </div>
+
+    <p class="status", id="angle">Angle: 0</p>
+
+    <div class="help_status">
+      <p class="status", id="manual"><u>Manual Mode: False </u></p>
+      <span class="status_help">M: ON <br> N: OFF <br> Shift: Up-gear <br> Ctrl: Down-gear </span>
+    </div>
+
+    <p class="status", id="gear">Current gear:</p>
+    <p class="status", id="distance">Distance: 0 m</p>
+    <p class="status", id="charge">Charge: 100%</p>
```

Comment:
There shouldn't be a comma between class and id inside of HTML elements

Reply:
Alright, I just deleted unnecessary commas